### DISCUSSION TITLE
I want to participate in GSoC 2023, need help to get started with open source.

### DISCUSSION THREAD

**Discussion by oyehanif** [oppia/oppia-android#4820] [Q&A (Other)] [answered]
Hey everyone, I am Hanif Shaikh , second year MCA student at R B Institute of management studies, Ahmedabad , India. I came across Liquid Galaxy on the GSOC organization page and i loved the ideas . I know Android development  related stuffs : Kotlin , MVVM, Firebase , APIs , Retrofit etc.. I need guidance to start working on this project.So, can you tell me more about this and guide me further about the same.

**Comment by YesShubham01** [comment]
Hey! @oyehanif 
Sorry this isn't a reply to help you.
I am a fellow newbie to open Source. I also need some guidance.
Can I get your mail or linked in?
So we can help each other during our learning?
Obviously, If you don't mind that.

**Reply by oyehanif** [reply]
Yay Sure. 
Email :- hanifshaikh8153@gmail.com
LinkedIn :- [Hanifshaikh](https://www.linkedin.com/in/hanif-shaikh-916282211/)

**Comment by MohitGupta121** [comment, marked as answer]
@oyehanif great to see your interest in open source but [oppia-android](https://github.com/oppia/oppia-android) is not participating in GSOC 2023. But you still contribute.

**Reply by BenHenning** [reply]
Following up on this: Oppia web is still planning to participate in GSoC this year, if you're interested in contributing to a web or backend project. See https://github.com/oppia/oppia/wiki/Google-Summer-of-Code-2022#getting-started for last year's instructions for an idea on how to get started (note that this year's instructions haven't yet been finalized & published, but you can keep an eye out via the oppia-gsoc-discuss@ mailing list).

**Reply by theMr17** [reply]
@MohitGupta121 Will [oppia-android](https://github.com/oppia/oppia-android) participate in GSOC 2024?

**Reply by MohitGupta121** [reply]
Yes @the-mr17 hopefully

**Comment by MohitGupta121** [comment]
Hey @oyehanif  if your issue is resolved please mark this discussion is resolved so that it's helpful to others in the community.
And if you need help then also feel free to ask.
